### PR TITLE
`make_pureDP_to_zCDP` constructor

### LINKED ISSUE #575: `make_pureDP_to_zCDP` constructor
Similar API as `make_zCDP_to_approxDP`. Can be used to compose mixtures of pure and zCDP measurements.

### PR DESCRIPTION
**Pull request by Shoeboxam**
Closes #575

### CODE REVIEW

File: rust/src/combinators/measure_cast/pureDP_to_zCDP/ffi.rs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+#[bootstrap(
+    features("contrib"),
+    dependencies("$get_dependencies(measurement)")
+)]
+/// Constructs a new output measurement where the output measure
+/// is casted from `MaxDivergence<QO>` to `ZeroConcentratedDivergence<QO>`.
+///
+/// # Citations
+/// - [BS16 Concentrated Differential Privacy: Simplifications, Extensions, and Lower Bounds](https://arxiv.org/pdf/1605.02065.pdf#subsection.3.2)
```
Comment (andrewvyrros) [inline rust/src/combinators/measure_cast/pureDP_to_zCDP/ffi.rs:19]: Should this be Section 3.1?

File: rust/src/combinators/measure_cast/pureDP_to_zCDP/mod.rs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+};
+
+#[cfg(feature = "ffi")]
+mod ffi;
+
+/// Constructs a new output measurement where the output measure
+/// is casted from `MaxDivergence<QO>` to `ZeroConcentratedDivergence<QO>`.
+///
+/// # Citations
+/// - [BS16 Concentrated Differential Privacy: Simplifications, Extensions, and Lower Bounds](https://arxiv.org/pdf/1605.02065.pdf#subsection.3.2)
```
Comment (andrewvyrros) [inline rust/src/combinators/measure_cast/pureDP_to_zCDP/mod.rs:15]: Should this be Section 3.1?

**Review by andrewvyrros — COMMENTED** [review]
Right on, just a question about the citation.

File: rust/src/combinators/measure_cast/pureDP_to_zCDP/ffi.rs (hunk 1 above)
Comment (Shoeboxam) [reply rust/src/combinators/measure_cast/pureDP_to_zCDP/ffi.rs]: Nice catch!

**Review by andrewvyrros — APPROVED** [review]

**Pull request merged**